src/HttSystematics_MSSMvsSMRun2.cc

```diff
@@ -69,7 +77,7 @@ void AddMSSMvsSMRun2Systematics(CombineHarvester &cb, bool jetfakes, bool embedd
       };
   std::vector<std::string> signals_VH = {
       // STXS stage 0
-      "WH125", "ZH125", "ttH125"};
+      "WH125", "Wh", "WH", "WH1", "ZH125", "Zh", "ZH", "ZH1", "ttH125"};
```

Comment:
Technically ttH125 not VH but does not matter here.

Reply:
yeah. We anyhow won't use it